types/gapi/index.d.ts

```diff
@@ -35,10 +35,20 @@ interface GoogleApiOAuth2TokenObject {
  */
 declare namespace gapi {
 
+    type LoadCallback = () => void | Function;
```

Comment:
Done. Removed `Function` and changed the fallback to `(...args: any[]) => void`.